### PR TITLE
implement all events page

### PR DESCRIPTION
**Pull request by Suuuuha**
related to #90

### CODE REVIEW

File: src/views/all_events.hbs [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div class="">
```
Comment (Karyum) [inline src/views/all_events.hbs]: empty class. not needed currently

File: src/views/all_events.hbs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  <h1>Upcoming Events</h1>
+  {{#each upcomingEvents}}
+  <span>{{this.date}}</span>
+  <div class="">
+    {{this.time}}
```
Comment (Karyum) [inline src/views/all_events.hbs]: be more specific with the key names.

File: src/views/all_events.hbs [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  </div>
+  <div class="">
+    {{this.title}}
+    {{this.location}}
+  </div>
+  <div class="">
```
Comment (Karyum) [inline src/views/all_events.hbs]: empty class

File: src/views/all_events.hbs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    <a href="{{this.rsvp}}">RSVP</a>
+  </div>
+  {{/each}}
+
+  <h2>Past Events</h2>
+  {{#each events}}
+  <span>{{this.date}}</span>
+  <div class="">
+    {{this.time}}
```
Comment (Karyum) [inline src/views/all_events.hbs]: both in upcoming and past events have the time key ? :maple_leaf:

Comment (Karyum) [inline src/views/all_events.hbs]: im not saying just be specific with the time key, also with the other keys

File: src/views/all_events.hbs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+  </div>
+  <div class="">
+    {{this.title}}
+    {{this.location}}
+  </div>
+  {{/each}}
+  </div>
+
```
Comment (Karyum) [inline src/views/all_events.hbs]: too many empty lines :maple_leaf:

File: src/views/all_events.hbs (hunk 1 above)
Comment (Suuuuha) [inline src/views/all_events.hbs]: left it empty for the tachyons later on

**Comment by esraajb** [comment]
@Karyum @Suuuuha fix travis please

**Pull request merged**